sdk/python/packages/flet/src/flet/controls/core/shader_mask.py

```diff
@@ -49,6 +47,6 @@ def main(page: ft.Page):
     """
 
     shader: Gradient
-    content: Optional[Control] = None
-    blend_mode: Optional[BlendMode] = None
+    content: OptionalControl = None
```

Comment:
I'm still not completely sold we should introduce a lot of "specific" types, like `OptionalControl` where you have to go there and see what that type really means. Moreover, starting from Python 3.11 it has nicer syntax: `Control | None`, so eventually we won't need that at all.